### ISSUE TITLE
Build dependencies on unavailable repository

### ISSUE THREAD

**Issue by crism** [pkp/xmlps#3]
```
crism@portmore:~/src/xmlps$ php composer.phar install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
  - Installing michaelthessel/parscit (110505b)
    Cloning 110505b
                                                                               
  [RuntimeException]                                                           
  Failed to clone git@github.com:MichaelThessel/ParsCit.git via git, https, s  
  sh protocols, aborting.                                                      
  - git://github.com/MichaelThessel/ParsCit.git                                
    Cloning into '/home/crism/src/xmlps/vendor/MichaelThessel/ParsCit'...      
    fatal: remote error:                                                       
      Repository not found.                                                    
 ```

**Comment by crism** [comment]
It looks like this is http://aye.comp.nus.edu.sg/parsCit/ which has an authoritative GitHub repo at http://www.github.com/knmnyn/parscit, though they note that that may not always be up to date. Michael’s copy seems to have gone away. Do we import from `knmnyn`, or make our own fork?

**Comment by jalperin** [comment]
Either one is fine, but since I don't anticipate we'll be making any changes to ParsCit in the near future, it is probably safe to use the official repo. Unless @axfelix knows of a reason.

**Comment by crism** [comment]
I think that’s resolved, but now I’m having trouble with `slm/queue-doctrine`—it seems fine in Packagist, but something else is going wrong. Could be a stability level problem; I’m still getting used to Conductor.
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package slm/queue-doctrine could not be found in any version, there may be a typo in the package name.
```

**Comment by axfelix** [comment]
I was able to duplicate -- if you check line 81 of the composer.json file, it looks like the last time we edited the deps, it was configured to fetch "0.3.x-dev" of slm/queue-doctrine, which no longer exists at https://packagist.org/packages/slm/queue-doctrine, presumably because they've released a stable version of 0.3.0 since then (look at the sidebar of that page).

I can successfully fetch it by changing the json file to reflect "0.3.0." Want to try that on your end? I haven't actually tested getting the whole stack running in a while, and I see you've already made a new git branch. If this works we can just push the dependency updates to master.
